cloudinary/__init__.py

```diff
@@ -174,6 +175,8 @@ def __init__(self):
             cloudinary_url = os.environ.get("CLOUDINARY_URL")
             parsed_url = self._parse_cloudinary_url(cloudinary_url)
             self._setup_from_parsed_url(parsed_url)
+        if not self.signature_algorithm:
```

Comment:
Keeping configuration defaults approach is different from what is used in the SDK right now.
Although it is a much clearer approach, I think it would be better to be consistent for now, and in new version of the SDK to do the refactoring.

For now, I suggest to align it with the rest of the code, and not set default here.

Reply:
From what I see, in Java SDK [there is a default value](https://github.com/cloudinary/cloudinary_java/pull/215/files#diff-a315a855b16da95dedc3e6ca39ad7e80b6b6572c44c35e815f9fd18ee88088e6R49) for signature algorithm, including [config update](https://github.com/cloudinary/cloudinary_java/pull/215/files#diff-a315a855b16da95dedc3e6ca39ad7e80b6b6572c44c35e815f9fd18ee88088e6R132). 
So it looks aligned. And also if we don't have default value here then we'll have it in multiple places in different functions, which will increase mistake chance in the future in case there is a need to change it.
Would you still prefer to align code with non-java SDK? May you please then provide a link to a preferable approach?

Reply:
OK, let's keep it like that.